src/main/resources/config.yml

```diff
@@ -75,6 +75,9 @@ island:
     # These set the minimum and maximum size of a name.
     min-length: 4
     max-length: 20
+  # Number of blocks to paste per tick when pasting a schem
+  # Smaller values will help reduce noticeable lag but will make pasting take longer
+  paste-speed: 1000
```

Comment:
I have no idea if this is a good speed or not.

Reply:
Hopefully we'll get some feedback about that.